### PR TITLE
Update getting started Android

### PR DESCRIPTION
**Pull request by dengdan154**
_Issue #, if available:_

_Description of changes:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by poojamat — APPROVED** [review]

**Review by div5yesh — CHANGES_REQUESTED** [review]

File: docs/start/getting-started/fragments/android/add-api.md [hunk 1]
```diff
@@ -1,48 +1,77 @@
...
-  ```java
+
+   ```java
   try {
       Amplify.addPlugin(new AWSApiPlugin());
       Amplify.addPlugin(new AWSDataStorePlugin());
       Amplify.configure(getApplicationContext());
-
-      Log.i("Tutorial", "Initialized Amplify");
+       Log.i("Tutorial", "Initialized Amplify");
```
Comment (div5yesh) [inline docs/start/getting-started/fragments/android/add-api.md:58]: Indentation fixes.

```suggestion
      Log.i("Tutorial", "Initialized Amplify");
```

File: docs/start/getting-started/fragments/android/add-api.md [hunk 2]
```diff
@@ -1,48 +1,77 @@
...
   } catch (AmplifyException failure) {
       Log.e("Tutorial", "Could not initialize Amplify", failure);
   }
-
-  Amplify.DataStore.observe(Todo.class,
+   Amplify.DataStore.observe(Todo.class,
```
Comment (div5yesh) [inline docs/start/getting-started/fragments/android/add-api.md:62]:
```suggestion
  Amplify.DataStore.observe(Todo.class,
```

File: docs/start/getting-started/fragments/android/add-api.md [hunk 3]
```diff
@@ -51,75 +80,38 @@ Now that your have DataStore persisting data locally, in the next step you'll co
   } catch (failure: AmplifyException) {
       Log.e("Tutorial", "Could not initialize Amplify", failure)
   }
-
-  Amplify.DataStore.observe(Todo::class.java,
+   Amplify.DataStore.observe(Todo::class.java,
```
Comment (div5yesh) [inline docs/start/getting-started/fragments/android/add-api.md:83]:
```suggestion
  Amplify.DataStore.observe(Todo::class.java,
```

**Review by div5yesh — APPROVED** [review]
Ship it 🚢

**Pull request merged**